style/rails/README.md

```diff
@@ -8,7 +8,8 @@ Rails
 * Name time columns (referring to a time of day with no date) with `_time`
   suffixes.
 * Name initializers for their gem name.
-* Order ActiveRecord associations alphabetically by attribute name.
+* Order ActiveRecord associations alphabetically by association type, then
+  attribute name. [Example][order-associations]
```

Comment:
What if we put the link on " Order ActiveRecord associations alphabetically" so the link flows as part of the text?